### PR TITLE
Correct missing ENV for deployment action for pre-release

### PR DESCRIPTION
**Pull request by rudyflores**
**What It Does**

Update env's missing for successful deployment, seems like it failed due to 2 missing env variables

**How to Test**

**Review Checklist**
I certify that I have:
- [ ] tested my changes
- [ ] added/updated automated tests
- [ ] updated the changelog
- [ ] followed the [contribution guidelines](https://github.com/zowe/zowe-cli/blob/master/CONTRIBUTING.md)

**Additional Comments**

### CODE REVIEW

**Review by zFernand0 — COMMENTED** [review]
let's hold off on updating these as the secrets have not being added yet...
any thoughts on the following secrets as Organization secrets?
```
VSCODE_VSCE_PUBLISHER_TOKEN
VSCODE_OVSX_PUBLISHER_TOKEN
```

**Comment by rudyflores** [comment]
> 

These ENV names sound good to me!

**Comment by zFernand0** [comment]
Quick update:
we got the tokens added:

image

File: .github/workflows/zowe-cli-plugin.yml [hunk 1]
```diff
@@ -139,3 +139,5 @@ jobs:
                   NPM_USERNAME: ${{ secrets.ARTIFACTORY_USERNAME }}
                   NPM_PASSWORD: ${{ secrets.ARTIFACTORY_PASSWORD }}
                   NPM_EMAIL: ${{ secrets.ZOWE_ROBOT_EMAIL }}
+                  OVSX_PAT: ${{ secrets.VSCODE_ZOWE_ROBOT_PUBLISH_TOKEN }}
+                  VSCE_PAT: ${{ secrets.VSCODE_PUBLISHER_TOKEN }}
```
Comment (zFernand0) [inline .github/workflows/zowe-cli-plugin.yml:143]: Could we change the secret ENV vars to the ones mentioned above?

**Review by zFernand0 — CHANGES_REQUESTED** [review]

**Review by traeok — APPROVED** [review]
LGTM, thanks @rudyflores!

**Review by zFernand0 — APPROVED** [review]

**Pull request merged**